archetypes/blog-post/index.md

```diff
@@ -34,6 +34,13 @@ authors:
 tags:
     - change-me
 
+
+# The social copy used to promote this post on Twitter and Linkedin. These
+# properties do not actually create the post and have no effect on the
+# generated blog page. They are here strictly for reference.
+social_twitter: "social copy to post on Twitter"
+social_linkedin: "social copy to post on Linkedin"
```

Comment:
You could also do this:

```suggestion
social:
    twitter: 
    linkedin: 
```

... And then leave these blank, as we do with the meta_desc.